src/Uno.UI.Runtime.WebAssembly/Xaml/UIElementWasmExtensions.cs

```diff
@@ -121,7 +121,7 @@ public static string ExecuteJavascript(this UIElement element, string jsCode)
 			var js = @$"
 (function(element) {{
 {jsCode}
-}})(Uno.UI.WindowManager.current.getView({element.HtmlId}));
+}})(Uno.UI.WindowManager.current.getView(""{element.HtmlId}""));
```

Comment:
My concern about this PR is if I missed any places where quotes are needed. I hope I didn't miss, and hope coverage is enough to fail the CI if anything is missing.